tljh-voila-gallery/setup.py

```diff
@@ -7,5 +7,11 @@
         "console_scripts": ["build-gallery-images = tljh_voila_gallery.build_images:main"]
     },
     packages=find_packages(),
-    include_package_data=True
+    include_package_data=True,
+    install_requires=[
+        # These get installed into the hub environment
+        'dockerspawner',
```

Comment:
Adding `repo2docker`:

```suggestion
        'dockerspawner',
        'repo2docker',
```